src/style.css

```diff
@@ -0,0 +1,164 @@
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+header {
+  width: 1200px;
+  height: 60px;
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-sizing: border-box;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  align-items: center;
+  margin: 0 auto;
+}
+
+header img {
+  width: 40px;
+  height: 40px;
+}
+
+nav {
+  left: 397px;
+  right: 50px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+  display: flex;
+  align-items: center;
+}
+
+a {
+  text-decoration: none;
+  color: #000;
+}
+
+a.is-active {
+  position: relative;
+}
+
+a:hover {
+  color: #00acdc;
+}
+
+a:not(:last-child) {
+  margin-right: 20px;
+}
+
+.is-active::after {
+  content: "";
+  width: 37px;
```

Comment:
this class must be adaptive.
If you add this class to elements that are larger, what we will see?